code/game/machinery/vr_console.dm

```diff
@@ -0,0 +1,222 @@
+/obj/machinery/vr_sleeper
+	name = "VR Sleeper"

```

Comment:
Should this be improper?

Reply:
VR is an acronym, but the sleeper probably should be, yeah